platforms/core-runtime/stdlib-java-extensions/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id("gradlebuild.distribution.api-java")
+    id("gradlebuild.distribution.uninstrumented.api-java")
```

Comment:
💭 This has to be `gradlebuild.distribution.uninstrumented.api-java` since this project is used in ann. process. And we want to avoid circular dependencies

Reply:
FTR, we only need this because this very low-level subproject contains API classes, which shouldn't be the case.

I think it would be good to add this same comment to the build script, too.

Reply:
Added a comment